### PR TITLE
UHM-6261, update Processed status and filter

### PR DESCRIPTION
**Pull request by cioan**
@mogoodrich , here are the changes for updating the PROCESSED status and filter. Thanks!

### CODE REVIEW

File: omod/src/main/webapp/pages/labtrackingOrderReadOnly.gsp [hunk 1]
```diff
@@ -1,99 +1,3 @@
-<div class="panel panel-default" ng-if="hasSpecimenCollection(order)">
```
Comment (mogoodrich) [inline omod/src/main/webapp/pages/labtrackingOrderReadOnly.gsp:1]: Is this just cleanup of something else?

File: omod/src/main/webapp/pages/labtrackingViewQueue.gsp [hunk 2]
```diff
@@ -60,12 +60,6 @@ ${patient?ui.includeFragment("coreapps", "patientHeader", [ patient: patient ])
                </label>
             </div>
 
-            <div class="col-md-6 text-left">
-               <label>
-                  <input id="urgentReview-search" type="checkbox"/>
-                  ${ui.message("labtrackingapp.orderdetails.urgentreviewlabel")}
-               </label>
-            </div>
```
Comment (mogoodrich) [inline omod/src/main/webapp/pages/labtrackingViewQueue.gsp:68]: Same question here...?

**Review by mogoodrich — APPROVED** [review]
@cioan  there were couple blocks of code that were removed that I flagged... if that was intentionally, looks good to me...

File: omod/src/main/webapp/pages/labtrackingOrderReadOnly.gsp (hunk 1 above)
Comment (cioan) [reply omod/src/main/webapp/pages/labtrackingOrderReadOnly.gsp:1]: Yes, just cleanup. We moved all the specimen details to the read only panel above.

File: omod/src/main/webapp/pages/labtrackingViewQueue.gsp (hunk 2 above)
Comment (cioan) [reply omod/src/main/webapp/pages/labtrackingViewQueue.gsp:68]: Yes, this was intentional. UHM-6185 says that urgentReview filter is optional (or less important than the suspectedCancer). It is easy to add it, but first I wanted to finish the mandatory items.

**Comment by cioan** [comment]
Thanks @mogoodrich !

**Pull request merged**